collect_app/src/main/java/org/odk/collect/android/activities/GeoTraceActivity.java

```diff
@@ -71,23 +74,42 @@
     private ImageButton pauseButton;
     private ImageButton backspaceButton;
 
+    private TextView locationStatus;
+    private TextView collectionStatus;
     private View traceSettingsView;
     private View polygonOrPolylineView;
 
     private AlertDialog traceSettingsDialog;
     private AlertDialog polygonOrPolylineDialog;
 
+    private static final int[] INTERVAL_OPTIONS = {
+        1, 5, 10, 20, 30, 60, 300, 600, 1200, 1800
+    };
+    private static final int DEFAULT_INTERVAL_INDEX = 3; // default is 20 seconds
+    private static final int MANUAL_RECORDING = 0;
+    private static final int AUTOMATIC_RECORDING = 1;
+
+    private static final int[] ACCURACY_THRESHOLD_OPTIONS = {
+        0, 3, 5, 10, 15, 20
+    };
+    private static final int DEFAULT_ACCURACY_THRESHOLD_INDEX = 3; // default is 10 meters
+    private static final String NO_LOCATION_COLOR = "#333333";
+    private static final String ACCEPTABLE_LOCATION_COLOR = "#337733";
+    private static final String UNACCEPTABLE_LOCATION_COLOR = "#773333";
```

Comment:
Can we add these to the bottom of `colors.xml`? There's not a ton there right now but I have an open PR that adds more there too.

Reply:
Yup, done!